### PR TITLE
BETA: Register jacksonmalone.is-a.dev

### PR DESCRIPTION
**Pull request by jacksonmalone**
Added `jacksonmalone.is-a.dev` using the [dashboard](https://manage.is-a.dev).

### CODE REVIEW

**Comment by jacksonmalone** [comment]
A few weeks ago I tried to create the domain "jacksonm.is-a.dev" using the dashboard, but I never could get the site to actually work. Everything I did didn't work and my domain just redirected me to "is-a.dev" every time. So, I am trying to create this domain with the CNAME set to <my-github-username>.github.io, because I saw that in the is-a.dev/docs about CNAME record in the domain structure section.

File: domains/jacksonm.json [hunk 1]
```diff
@@ -1,14 +1,10 @@
...
         "email": "jrm9918@gmail.com"
     },
     "record": {
-        "A": [
-            "217.174.245.249",
-            "51.161.54.161"
-            ],
-        "MX": ["mail.is-a.dev"],
-        "TXT": "v=spf1 mx a:mail.is-a.dev ~all"
+        "CNAME": "jacksonmalone.github.io"
```
Comment (Stef-00012) [inline domains/jacksonm.json:8]: I cannot reach your site (404 page)

File: domains/jacksonmalone.json [hunk 2]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "jacksonmalone",
+        "email": "jrm9918@gmail.com"
+    },
+    "record": {
+        "CNAME": "jacksonmalone.github.io"
```
Comment (Stef-00012) [inline domains/jacksonmalone.json:7]: I cannot reach your site (404 page)

**Review by Stef-DP — CHANGES_REQUESTED** [review]

File: CNAME
Comment (Stef-00012) [inline CNAME:1]: You can't add files i the root directory

**Review by Stef-DP — CHANGES_REQUESTED** [review]

File: CNAME
Comment (jacksonmalone) [reply CNAME:1]: I'm confused. All I did was use the dashboard to do this automatically. I thought all I needed to do is make a json file in the domains directory, I didn't try to add files to the root directory. If there is anything I am supposed to be doing to make this work, I would greatly appreciate any help here.

Comment (jacksonmalone) [reply CNAME:1]: I created my github pages at jacksonmalone.github.io and it is redirecting to jackson.is-a.dev but then it gives an error and goes to is-a.dev .

**Pull request closed without merging**